spec/index.html

```diff
@@ -227,7 +227,7 @@ <h2>Transmission issues using CSV and TSV Formats</h2>
     <h2>CSV — Comma Separated values</h2>
     <p>In the SPARQL Results CSV Format, the results table is serialized as one line listing the variables in
       the results, using the CSV header line, followed by one line for each query solution. (Note: a line may end up
-      split by newlines in the data). Values in the results are strings for URIs, literals and blank nodes, together
+      split by newlines in the data). Values in the results are strings for URIs, literals, and blank nodes, together
```

Comment:
"for URIs, literals, and blank nodes" => "for URIs, literals, and for blank nodes" (or don't use an Oxford/Harvard/... comma).

Reply:
The Oxford/Harvard/terminal comma reduces possible reader confusion to virtually zero (I've yet to see an example where it _increases_ confusion in a reader), while omitting that terminal comma almost always increases possible if not actual confusion.

`for URIs, literals, and for blank nodes` doesn't make sense to me at all. It logically expands to `for URIs, for literals, and for blank nodes`, which logically reduces (by removing redundancy) to `for URIs, literals, and blank nodes` (as I originally suggested) or perhaps `for blank nodes, URIs, and literals` (if I'm guessing right about your concern with my suggestion, though I think that implies that the `blank` noun modifier applies to `URIs` and `literals` as well as `nodes`, because that concept has a compound name).

(The horse has long since left the barn on `blank nodes`, as a compound name for those things. They might have better been called `pronodes` a la `pronouns`, which is the primary if not only role they serve.)

Reply:
It is the part after that reads badly to me. The 'together' groups with the blank nodes.

"..., and blank nodes, together with numbers". 

May be "together" => "and also"

Reply:
Ah, I see. Lines 230 and 231 must be considered at once (while I saw your initial comment in this thread as only on line 230). I see two possible interpretations of the full sentence. See new suggestions below, one of which should be acceptable.